app.css

```diff
@@ -0,0 +1,228 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 90%;
+  margin: 0 auto;
+}
+
+.headline {
+  background-color: #2a8bb5;
+  overflow: auto;
+  padding: 0 2rem;
+}
+
+.headline__title {
+  background-color: #2a8bb5;
+  float: left;
+  margin: 2rem;
```

Comment:
Don't use left and right margin here, it will make the nav easier to line up with the lower content.